learning_journal/models.py

```diff
@@ -25,3 +28,28 @@ class MyModel(Base):
     value = Column(Integer)
 
 Index('my_index', MyModel.name, unique=True, mysql_length=255)
+
+class Entry(Base):
+    __tablename__ = 'entries'
+    id = Column(Integer, primary_key=True)
+    title = Column(Unicode(255), unique=True, nullable=False)
+    body = Column(Text)
+    created = Column(DateTime, default=datetime.datetime.utcnow)
+    edited = Column(DateTime, default=datetime.datetime.utcnow)
+    
+    @classmethod
+    def all(class_):
+        """Return a query of all entries."""
+        Entry = class_
+        q = DBSession.query(Entry)
```

Comment:
Please be careful in using one-letter variable names.  The letter `q` in debuggers is generally a synonym for `quit`, which exits the debugger without continuing the process.  You probably don't want to try to see the value of this variable using that letter as its name when debugging.  It won't end happily for you.